cppcache/include/geode/Pool.hpp

```diff
@@ -20,8 +15,16 @@
  * limitations under the License.
  */
 
+#pragma once
+
+#ifndef GEODE_POOL_H_
+#define GEODE_POOL_H_
+
```

Comment:
Technically I believe #pragma once and #ifndef GEODE_POOL_H_, but the #ifndef GEODE_POOL_H_  is more widely supported. Why are we  doubling up everywhere?

Reply:
`#pragma once` is not a C++ standard and is not supported in all compilers.